README.md

```diff
@@ -9,7 +9,7 @@ This repository contains the code needed to enable Intel(R) nGraph(TM) Compiler
 runtime engine for TensorFlow. Use it to speed up your TensorFlow training and 
 inference workloads. The nGraph Library and runtime suite can also be used to 
 customize and deploy Deep Learning inference models that will "just work" with 
-a variety of nGraph-enabled backends: CPU, GPU, and custom silicon like the 
+a variety of nGraph-enabled backends: CPU, and custom silicon like the 
```

Comment:
Can you also remove the line down in this file where it says `--build_intelgpu_backend`?